components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/manage_service_providers.jsp

```diff
@@ -207,24 +208,24 @@
                             if (sp != null) {
                 %>
                 <tr>
-                    <td style="width:20%"><%=sp.getIssuer()%>
+                    <td style="width:20%"><%=Encode.forHtml(sp.getIssuer())%>
                     </td>
-                    <td><%=sp.getDefaultAssertionConsumerUrl()%>
+                    <td><%=Encode.forHtml(sp.getDefaultAssertionConsumerUrl())%>
                     </td>
-                    <td style="width:10%"><%=sp.getCertAlias() == null ? "" : sp.getCertAlias()%>
+                    <td style="width:10%"><%=sp.getCertAlias() == null ? "" : Encode.forHtml(sp.getCertAlias())%>
                     </td>
                     <td style="width:15%"><%=sp.getAttributeConsumingServiceIndex() == null
                                              ? ""
-                                             : sp.getAttributeConsumingServiceIndex()%>
+                                             : Encode.forHtml(sp.getAttributeConsumingServiceIndex())%>
                     </td>
                     <td style="width:200px;white-space:nowrap;"><a title="Edit Service Providers"
-                                                                   onclick="edit('<%=sp.getIssuer()%>');return false;"
+                                                                   onclick="edit('<%=Encode.forJavaScriptAttribute(sp.getIssuer())%>');return false;"
```

Comment:
Here the passed parameter is used in a query string. Therefore first uri encode and then javascript attribute encode